### PR TITLE
faq: update MPI/PMI debug info

### PR DESCRIPTION
**Pull request by garlick**
Problem: some of the mpi/pmi info in the FAQ is outdated.
    
Update pmi and mpi options.

### CODE REVIEW

File: faqs.rst [hunk 1]
```diff
@@ -16,33 +16,33 @@ Background
 What is Flux Framework?
 =======================
 
-Flux is a flexible framework for resource management, built for your site. 
-The core framework here consists of a suite of projects, tools, and libraries which may be used to build site-custom resource managers for 
+Flux is a flexible framework for resource management, built for your site.
+The core framework here consists of a suite of projects, tools, and libraries which may be used to build site-custom resource managers for
```
Comment (vsoch) [inline faqs.rst:20]:
```suggestion
The core framework here consists of a suite of projects, tools, and libraries that may be used to build site-custom resource managers for
```

**Review by vsoch — COMMENTED** [review]
@garlick if we add a pre-commit setup here, it's very easy to handle cleaning up spaces at the end of lines! Let me know if that is something you'd be interested in.

File: faqs.rst (hunk 1 above)
Comment (garlick) [reply faqs.rst]: Just added a commit to make the word change.  I'm sort of ambivalent on the pre-commit hook.  Could we table that for now?

**Review by grondo — APPROVED** [review]
Looks good, thanks!

**Pull request merged**